.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - feature/[a-z]+-[0-9]+/dev-*
+  workflow_dispatch:
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.1',
+            '7.4'
```

Comment:
This should be min/max with latest 8.1 now

Reply:
it wasn't tested on php 8+, so I'm not sure it will work

Reply:
changed to 8.1

Reply:
Any reason why we are not going 7.3+?
Then also testing is easier, e.g. PHPUnit 9+ only.

Reply:
Well, it works with php 7.1 as well, does it make sense to limit the PHP version with 7.3+?

Reply:
We dont technically support any tool below this for the reasons stated above (not even code sniffer works here anymore which is always at least 1 minor behind).

Reply:
Note: Also propel itself will now be 7.3+ shortly.